stock_account_quantity_history_location/__manifest__.py

```diff
@@ -19,6 +19,6 @@
     'data': [
         'wizards/stock_quantity_history.xml',
     ],
-    'auto_install': True,
+    'auto_install': False,
```

Comment:
This is correct to be auto-installed when both dependencies are there

Reply:
ok, I'll keep it like it was. Only this module maybe isn't pertinent anymore, as it doesn't solve anything that cannot provide this one...

Reply:
Then remove it. cc @JordiBForgeFlow

Reply:
Sorry, I was wrong. There's at least a view that need to be covered. So I just keep it as it was.